docs/dev/cli/virtual-usb/start-server.md

```diff
@@ -4,53 +4,54 @@ title: "Virtual USB CLI: Start the Server"
 sidebar_label: Start Server
 ---
 
-__Full Example__:
+The `server` command connects your local machine to the Sauce Labs Data Center where your Real Device Cloud private devices are hosted.
```

Comment:
Can you also mention that this commands needs to be started in a separate terminal and will run the whole time when they connect with different devices. All connection logs, even from the devices, can be find in this terminal

Reply:
done! added a short note here, and added a more elaborate note in the main vUSB guide